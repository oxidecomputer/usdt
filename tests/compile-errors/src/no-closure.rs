#![feature(asm)]

usdt::dtrace_provider!("../../../tests/compile-errors/providers/type-mismatch.d");

fn main() {
    let arg: u8 = 0;
    mismatch_bad!(arg);
}
