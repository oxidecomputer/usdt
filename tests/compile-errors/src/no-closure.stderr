error: proc macro panicked
  --> src/no-closure.rs:15:1
   |
15 | usdt::dtrace_provider!("../../../tests/compile-errors/providers/type-mismatch.d");
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: Could not read D source file "../../../tests/compile-errors/providers/type-mismatch.d" in "$WORKSPACE/target/tests/trybuild/compile-errors"

error[E0433]: failed to resolve: use of undeclared crate or module `mismatch`
  --> src/no-closure.rs:19:5
   |
19 |     mismatch::bad!(arg);
   |     ^^^^^^^^ use of undeclared crate or module `mismatch`
